### PR TITLE
Add staging Kubernetes config

### PR DESCRIPTION
**Pull request by adammcmaster**

### CODE REVIEW

**Comment by adammcmaster** [comment]
As far as I can tell, this app only uses Redis so that each instance of it can discover the other instances. See #35. If that's the case, we won't need to migrate the existing Redis data, and I'm also not sure it's worth the cost of setting up a managed redis server for it. My plan is to just add a redis deployment to Kubernetes.

@camallen does that sound OK to you?

**Comment by camallen** [comment]
> As far as I can tell, this app only uses Redis so that each instance of it can discover the other instances. See #35. If that's the case, we won't need to migrate the existing Redis data, and I'm also not sure it's worth the cost of setting up a managed redis server for it. My plan is to just add a redis deployment to Kubernetes.
> 
> @camallen does that sound OK to you?

We currently share a redis service with the main API redis service. We do this as the panoptes API is where the cellect client is called. However, we do use a different database on that redis service via the `/1` suffix, e.g. sidekiq uses `db=0` and attention uses `db=1` so their data is isolated from each other. `ATTENTION_REDIS_URL=redis://panoptes-redis.zooniverse.org:6379/1`

I'd say we don't have to migrate the data for attention (and most likely not for panoptes either if we drain it correctly). We do need that redis server to be low latency / available as it's in the subject selection code path. 

That said the plan is to move off cellect (currently only 1 or two projects, https://cellect.zooniverse.org/stats) so I'd be ok with a self managed redis server as well.

File: kubernetes/deployment-staging.tmpl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+kind: Deployment
+metadata:
+  name: cellect-staging-app
+  labels:
+    app: cellect-staging-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cellect-staging-app
```
Comment (zwolf) [inline kubernetes/deployment-staging.tmpl]: Is this label going to conflict when (if?) we bring Cellect over to the azure k8s cluster?

Comment (adammcmaster) [reply kubernetes/deployment-staging.tmpl]: The other Cellect repo isn't deployed anywhere – this one is built on top of that one and just adds the Panoptes DB connection (unless I've misunderstood).

Comment (camallen) [reply kubernetes/deployment-staging.tmpl]: that is correct - this repo includes the cellect-server gem https://github.com/zooniverse/cellect_panoptes/blob/c6057b99330affc226c6437c99215dc90b6f6b0f/Gemfile#L9

and as it's a rack app it's run via https://github.com/zooniverse/cellect_panoptes/blob/c6057b99330affc226c6437c99215dc90b6f6b0f/config.ru#L22

**Comment by adammcmaster** [comment]
I've added a Redis service. I deliberately haven't given Redis a persistent volume claim since its data can be ephemeral.

**Pull request merged**